javascript-functions/javascript-functions-quiz-notes.md

```diff
@@ -0,0 +1,47 @@
+# javascript-functions-quiz-notes
+
+## Quiz Questions
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+- What is a function in JavaScript?
+
+- Describe the parts of a function **definition**.
+
+- Describe the parts of a function **call**.
+
+- When comparing them side-by-side, what are the differences between a function **call** and a function **definition**?
+
+- What is the difference between a **parameter** and an **argument**?
+
+- Why are function **parameters** useful?
+
+- What two effects does a `return` statement have on the behavior of a function?
+
```

Comment:
Don't forget to answer the quiz questions!

Reply:
I'm sorry i forgot to save before sending!